[python-mlflow] unable to resolve npm modules/deps

```
==> Starting build()...
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: js@0.1.0
npm ERR! Found: eslint@7.32.0
npm ERR! node_modules/eslint
npm ERR!   eslint@"^7.23.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^4.8.0" from eslint-config-airbnb-base@12.0.2
npm ERR! node_modules/eslint-config-airbnb-base
npm ERR!   eslint-config-airbnb-base@"~12.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/acxz/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/acxz/.npm/_logs/2022-05-17T20_08_30_473Z-debug-0.log
==> ERROR: A failure occurred in build().
    Aborting...
```

--- a/python-mlflow/.SRCINFO
+++ b/python-mlflow/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = python-mlflow
 	pkgdesc = An open source platform for the machine learning lifecycle
 	pkgver = 2.1.1
-	pkgrel = 1
+	pkgrel = 2
 	url = https://mlflow.org
 	arch = x86_64
 	license = Apache-2.0
@@ -10,7 +10,7 @@ pkgbase = python-mlflow
 	makedepends = python-installer
 	makedepends = python-wheel
 	makedepends = python-setuptools
-	makedepends = npm
+	makedepends = yarn
 	depends = python
 	depends = python-alembic
 	depends = python-docker

--- a/python-mlflow/PKGBUILD
+++ b/python-mlflow/PKGBUILD
@@ -2,7 +2,7 @@
 
 pkgname=python-mlflow
 pkgver=2.1.1
-pkgrel=1
+pkgrel=2
 pkgdesc='An open source platform for the machine learning lifecycle'
 arch=('x86_64')
 url='https://mlflow.org'
@@ -33,7 +33,7 @@ optdepends=('python-scikit-learn: for mlflow[extras, pipelines]'
             'python-aliyunstoreplugin: for mlflow[aliyun-oss]'
             )
 makedepends=('python' 'python-build' 'python-installer' 'python-wheel'
-             'python-setuptools' 'npm' 'yarn')
+             'python-setuptools' 'yarn')
 source=("$pkgname-$pkgver::https://github.com/mlflow/mlflow/archive/v$pkgver.tar.gz")
 sha256sums=('1d9933a658f8a31ba4902a5d3cd247c3e28acc2930ceac224c0266b3e737e253')
 

--- a/python-mlflow/PKGBUILD
+++ b/python-mlflow/PKGBUILD
@@ -33,17 +33,18 @@ optdepends=('python-scikit-learn: for mlflow[extras, pipelines]'
             'python-aliyunstoreplugin: for mlflow[aliyun-oss]'
             )
 makedepends=('python' 'python-build' 'python-installer' 'python-wheel'
-             'python-setuptools' 'npm')
+             'python-setuptools' 'npm' 'yarn')
 source=("$pkgname-$pkgver::https://github.com/mlflow/mlflow/archive/v$pkgver.tar.gz")
 sha256sums=('1d9933a658f8a31ba4902a5d3cd247c3e28acc2930ceac224c0266b3e737e253')
 
 _pkgname=mlflow
 
 build() {
+  export NODE_OPTIONS=--openssl-legacy-provider
   cd "${srcdir}/${_pkgname}-${pkgver}"
   cd mlflow/server/js
-  npm install
-  npm run build
+  yarn install
+  yarn run build
 
   cd "${srcdir}/${_pkgname}-${pkgver}"
   python -m build --wheel --no-isolation

